src/input-container/input-container.scss

```diff
@@ -131,11 +79,7 @@
       > .iui-checkbox-wrapper,
       > .iui-radio-wrapper,
       > .iui-toggle-switch-wrapper {
-        margin-right: $iui-m;
-
-        @supports (gap: $iui-m) {
-          margin-right: 0;
-        }
+        margin-right: 0;
       }
 
       &:not(:last-child) {
```

Comment:
I think this should also be removed

Reply:
I tried removing this one and it lead to some visual changes. I'll look into it a little further

Reply:
This use of `margin-right` is to separate `iui-input-group` from `iui-end-icon`, so it must be kept.